dotcom-rendering/cypress/integration/parallel-4/signedin.spec.js

```diff
@@ -28,7 +28,11 @@ describe('Signed in readers', function () {
 			return false;
 		});
 		// Mock call to 'profile/me'
-		cy.intercept('GET', '**/profile/me', profileResponse);
+		cy.intercept(
+			'GET',
+			'**/profile/me?strict_sanctions_check=false',
+			profileResponse,
+		);
```

Comment:
This is a take it or leave it comment, so I'll leave it up to you if you think it's better or not, but it may be more in keeping with the intentions of the test to use a wildcard here? I.e.:

```suggestion
		cy.intercept(
			'GET',
			'**/profile/me**',
			profileResponse,
		);
```

The downside is it will also match on `profile/me/special` but I don't expect these endpoints to appear.